src/config/types.ts

```diff
@@ -253,6 +273,9 @@ export class CommonLayerProps {
 
   @optional
   featureInfoProps?: { [key: string]: FeatureInfoProps };
+
+  @optional
+  menuGroup?: MenuGroup;
```

Comment:
Add a docstring to explain how this is used

Reply:
I meant, actually explain ;)

-> and give an example of formatted menuGroup

Reply:
is this ok?

![image](https://user-images.githubusercontent.com/25983260/166200584-4334fcd6-79ee-4944-952a-8c0af97f0e56.png)